### PR TITLE
[Validator] added missing translation for UK validator

### PR DESCRIPTION
**Pull request by andrii-boiko**
| Q             | A
| ------------- | ---
| Branch?       | 3.4 
| Bug fix?      | yes
| New feature?  |no 
| BC breaks?    | no  
| Deprecations? |no 
| Tests pass?   | yes    
| Fixed tickets | #30194
| License       | MIT

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf [hunk 1]
```diff
@@ -318,6 +318,18 @@
...
+                <source>This is not a valid UUID.</source>
+                <target>Це не валідне значення UUID.</target>
+            </trans-unit>
+            <trans-unit id=“84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Це значення повинне бути кратним {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id=“85”>
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Банківський код (BIC) не пов’язаний із міжнародним номером банківського рахунку (IBAN) {{ iban }}.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf]: Google Translate suggest me that `пов’язаний` should be spelled -> `пов'язаний`

**Comment by fabpot** [comment]
Thank you @Vynder.

File: src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf [hunk 2]
```diff
@@ -318,6 +318,18 @@
                 <source>Error</source>
                 <target>Помилка</target>
             </trans-unit>
+            <trans-unit id=“83”>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf]: There's a minor issue with the quotes used in `id="83"`, `id=“84"` and `id=“85”`. You have used "literary quotes" (`“` and `”`) but we must use "programming quotes" (`"`). Thanks!

**Review by javiereguiluz — COMMENTED** [review]
Thanks!

File: src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf (hunk 2 above)
Comment (fabpot) [reply src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf]: Good catch, fixed in 3b8893fd92

**Pull request merged**